plugins/techdocs-backend/package.json

```diff
@@ -59,6 +59,7 @@
   },
   "dependencies": {
     "@backstage/backend-common": "workspace:^",
+    "@backstage/backend-defaults": "workspace:^",
```

Comment:
Nop this dependency is not alright 😅, prolly fixed by switching to the cache service for `createRouter` instead

Reply:
Change reverted in the create router options.